doc/api/async_hooks.md

```diff
@@ -712,6 +712,75 @@ never be called.
 * Returns: {number} The same `triggerAsyncId` that is passed to the
 `AsyncResource` constructor.
 
+## Class: AsyncContext
+
+This class is used to create contexts that can be used through the event loop.
+
+### Example
+
+Let's build a logger that will always know the current HTTP request and use
+it to display enhanced logs without needing to explicitly pass the current
+HTTP request to it.
+
+```js
+const { AsyncContext } = require('async_storage');
```

Comment:
This should now be const { AsyncContext } = require('util');

Reply:
Ack, good catch